notebooks/vertex_genai/solutions/gen_ai_evaluation_service.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>Also good to add API document that shows possible string values for metrics (not explanation of each)?</p><p> </p><p><a href="https://cloud.google.com/vertex-ai/generative-ai/docs/reference/python/latest/vertexai.preview.evaluation.EvalTask#vertexai_preview_evaluation_EvalTask" target="_blank">https://cloud.google.com/vertex-ai/generative-ai/docs/reference/python/latest/vertexai.preview.evaluation.EvalTask#vertexai_preview_evaluation_EvalTask</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Nice suggestion. Thanks! </p>